### ISSUE TITLE
Bright white flash when returning focus to the steam client window

### ISSUE THREAD

**Issue by kaarelen** [ValveSoftware/steam-for-linux#10288]
#### Your system information

* Steam client version (build number or date): 
Steam Version:  1701289036
Steam Client Build Date:  Wed, Nov 29 8:59 PM UTC -08:00
Steam Web Build Date:  Wed, Nov 29 8:03 PM UTC -08:00

* Distribution (e.g. Ubuntu): Arch
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes, system fully upgraded
* Steam Logs: [steam-logs.tar.gz](https://github.com/ValveSoftware/steam-for-linux/files/13633718/steam-logs.tar.gz)
* GPU: Nvidia/Intel/AMD Nvidia (prop driver)

#### Please describe your issue in as much detail as possible:
What happens: after returning focus to the steam client window, it flashes with bright white screen, which feels quite painful for eyes, in a dark environment.
What I'm expecting: please just drop a dark rectangle into the client window when webcontent isn't rendered. (Chromium doing similar thing if dark theme chosen, for example)

#### Steps for reproducing this issue:

1. Open steam
2. Change workspace
3. Return to workspace with opened steam client
4. If you are sitting in the dark, close your eyes, or you will lose them by looking at supernova

Regards,
Tim

**Comment by kisak-valve** [comment]
Hello @kaarelen, can you check if `Steam` -> `Settings` -> `Interface` -> `Enable GPU accelerated rendering in web views` is turned on, and if it isn't, if enabling it improves the Steam client's behavior?

**Comment by kaarelen** [comment]
@kisak-valve yeah, you are right, turning on gpu accel fixes it. I turned it off while in client was memory leak bug, but right now it fixed so I can set this setting back to "on".

So for me, it's definitely good enough, and we can close the issue if you think that it's okay.

Thanks for your help!

**Comment by kaarelen** [comment]
I rushed too fast.
Memory leak still an issue, so GPU acceleration, unfortunately, still in unusable state, at least on nvidia.

**Comment by ojsl1** [comment]
This is still an issue on both current steam stable and beta. Enabling the gpu acceleration setting fixes the main client window, but other windows like Friends, Settings, and Screenshots still keep flashing.

steam stable: 1773426488
steam beta: 1773983034
manjaro Linux 6.18.19
KDE Plasma 6.6.3
AMD 7900XTX, Mesa 26.0.3

**Comment by fuzxi** [comment]
I am having this issue on i3wm. Enabling GPU accelerated rendering in web views fixes it, however this causes the library view to stop visually updating after changing workspaces. The library UI can still be interacted with, and other views, like the store, work normally apart from part of the library rendering near the top of the window. Starting steam with the `-system-composer` flag fixes the library bug but causes the white flash issue to return.

**Comment by SPREEKDOS** [comment]
The white flash is still an issue in the stable steam client (Version: 1782866176) at initial start and when focus is returned, didn't try the beta version but its probably not fixed there, this issue has been occurring since I bought an nvidia gpu about 2 years ago, so it has something to do with nvidia, but I think steam should at least mitigate the issue, its still annoying.

Here is a video output illustrating the issue:

https://github.com/user-attachments/assets/c21d192b-1c7c-4f09-a75a-ea6582188b21

<details>

<summary>System infomation</summary>

```
Operating System:
Linux Mint 22.3 (64 bit)
Kernel Name: Linux
Kernel Version: 6.17.0-35-generic
X Server Vendor: The X.Org Foundation
X Server Release: 12101011
X Window Manager: Xfwm4
Steam Runtime Version: steam-runtime_1.0.20260525.238139
Client Information:
Version: 1782866176
Browser GPU Acceleration Status: Disabled
Browser Canvas: Unavailable
Browser Canvas out-of-process rasterization: Disabled
Browser Direct Rendering Display 
Video Card:
Driver: NVIDIA Corporation NVIDIA GeForce RTX 3060/PCIe/SSE2
Driver Version: 4.6.0 NVIDIA 595.71.05
OpenGL Version: 4.6
Desktop Color Depth: 24 bits per pixel
Monitor Refresh Rate: 60 Hz
VendorID: 0x10de
DeviceID: 0x2504
Revision Not Detected
Number of Monitors: 2
Number of Logical Video Cards: 1
Primary Display Resolution: 1920 x 1080
Desktop Resolution: 4480 x 1440
Primary Display Size: 20.75" x 11.65" (23.78" diag), 52.7cm x 29.6cm (60.4cm diag)
Primary Bus: PCI Express 16x
Primary VRAM: 12288 MB
```

</details>
